src/main/java/gg/projecteden/nexus/models/freeze/Freeze.java

```diff
@@ -122,4 +124,21 @@ private void unmount() {
 				player.getVehicle().remove();
 	}
 
+	// There may be a better way of doing this, but I'm unsure.
+	public boolean isInArea() {
+		final int xOffset = 3;
+		final int yOffset = 2;
+
+		if (getPlayer().getLocation().getX() > location.getX() + xOffset)
+			return false;
+		else if (getPlayer().getLocation().getX() < location.getX() - xOffset)
+			return false;
+		else if (getPlayer().getLocation().getY() > location.getY() + yOffset)
+			return false;
+		else if (getPlayer().getLocation().getY() < location.getY() - yOffset)
+			return false;
+		else
+			return true;
+	}
```

Comment:
```suggestion
	private static final int MAX_DISTANCE = 3;

	public boolean isInArea() {
		return getOnlinePlayer().getLocation().distance(location) >= MAX_DISTANCE;
	}
```

Reply:
Is there a point in having MAX_DISTANCE be in the class scope?

Reply:
just to have it be static, ie only initialized once. there's no need to create the variable each time we check the distance, its never going to change.

Reply:
given the code was simplified to one line, `MAX_DISTANCE` could probably be inlined since it's only used once, it's a trade-off between readability and memory efficiency now. i think inlining it would be fine if you prefer that, although i personally like having static vars as it makes it easier/faster to tweak numbers compared to having magic numbers sprinkled throughout a class ¯\\\_(ツ)_/¯

Reply:
Ah gotcha, fair enough! tyty

Reply:
Yes, magic numbers should be avoided. Always assign a name to them to explain what they do/why they exist

Reply:
Missed Lexi's comment before posting/resolving, whoops.

Yeah, for this case it's probably not needed for MAX_DISTANCE to be a variable, but the readability benefit is quite nice. I'll keep it the way it is now though for the readability, and I'll remember to be more consistent with that moving forward 👍